osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -162,14 +158,6 @@ protected override void LogoSuspending(OsuLogo logo)
             seq.OnAbort(_ => buttons.SetOsuLogo(null));
         }
 
-        private void beatmap_ValueChanged(ValueChangedEvent<WorkingBeatmap> e)
-        {
-            if (!this.IsCurrentScreen())
```

Comment:
you removed this conditional. it's going to be triggering changes even when the background isn't current?